app/models/container_group.rb

```diff
@@ -37,6 +38,18 @@ def ready_condition_status
     ready_condition.try(:status) || 'None'
   end
 
+  def containers_states_summary
+    summary = Hash.new(0)
+    containers.each { |container| summary[container.state.to_sym] += 1 }
+    summary
+  end
+
+  def running_containers_summary
+    containers = self.containers.to_a
+    running = containers.count { |container| container.state.eql? 'running' }
```

Comment:
@tenderlove @matthewd do you have anything more efficient for this? (Maybe a `count` query?)